apps/examples/e2e/tests/fixtures/fixtures.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react-hooks/rules-of-hooks */
```

Comment:
Out of curiosity, why do we need this? Is it just the `use` name that's causing it?

Reply:
oh interesting, you might be right actually - lemme take a look at renaming the `use`

Reply:
ah yeah, i'll rename here, good callout